mesonbuild/mdist.py

```diff
@@ -233,7 +285,11 @@ def determine_archives_to_generate(options):
         sys.exit('No archive types specified.')
     return result
 
+
 def run(options):
+    '''
+    this is a runner function for 'meson dist'
```

Comment:
You don't need to say "this is a", just say `runner function for 'meson dist'`. Same for all the rest.

Reply:
Sorry, my brain was on autopilot, will fix this.